.mailmap

```diff
@@ -223,6 +223,7 @@ Leeseean Chiu <leeseean@qq.com>
 Luke Bayes <lbayes@patternpark.com>
 Lydia Kats <llkats@gmail.com> Lydia Katsamberis <llkats@gmail.com>
 Maciej Małecki <maciej.malecki@notimplemented.org> <me@mmalecki.com>
+MaleDong <maledong_github@outlook.com>
```

Comment:
https://git-scm.com/docs/git-check-mailmap#_mapping_authors

Based on the information you have provided, I think this should be:
```suggestion
MaleDong <maledong_github@outlook.com> <maledong_private@qq.com>
```

The way to check would be to rerun [`tools/update-authors.js`](https://github.com/nodejs/node/blob/master/tools/update-authors.js) and check if the changed `AUTHORS` includes your old email or new one.